woocommerce/wc-admin
# Reduce net order total by refunded amount less tax and shipping refunds

Fixes #1555 

Reflects refunds in net total without refunds towards shipping or tax.

### Screenshots
<img width="811" alt="screen shot 2019-02-18 at 5 33 44 pm" src="https://user-images.githubusercontent.com/10561050/52941296-6513e800-33a3-11e9-8809-681d4aa62a98.png">
<img width="632" alt="screen shot 2019-02-18 at 5 33 26 pm" src="https://user-images.githubusercontent.com/10561050/52941297-65ac7e80-33a3-11e9-9a04-6d1e952455a1.png">

### Detailed test instructions:

1.  Create an order with shipping and taxes.
2.  Refund the line items on that order with separate amounts for shipping and tax.
3.  Note the refund amount is correct (does not include refunds towards tax or shipping).
4.  Make a manual refund without entering a line item number.
5.  Make sure the refund amount is correct.
6.  Make a manual refund greater than the possible net_total.
7.  Make sure `net_total` is 0.
